gpflow/models/sgpmc.py

```diff
@@ -86,7 +87,13 @@ def __init__(
             loc=to_default_float(0.0), scale=to_default_float(1.0)
         )
 
-    def log_likelihood(self, *args, **kwargs) -> tf.Tensor:
+    def log_posterior_density(self) -> tf.Tensor:
+        return self.log_conditional_likelihood_lower_bound() + self.log_prior_density()
```

Comment:
isn't this redundant?

Reply:
for the MCMC models I think it's good to be explicit